src/core/qfield.h

```diff
@@ -0,0 +1,21 @@
+#ifndef QFIELD_H
```

Comment:
License headers

Reply:
Aaah, sorry, should add this to the commit hooks :D